tests/test_spelling.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
```

Comment:
@erickwilder, yes this is also renaming, but I messed something up with the commit --amend or maybe you are not looking at the latest version. When I rename is it necessary, to explicitly `rm` the old file?